cnf-certification-test/accesscontrol/securitycontextcontainer/securitycontextcontainer.go

```diff
@@ -0,0 +1,495 @@
+		result = false
+	}
+	// HostIPC is true if the HostIPC field is set to true, false otherwise.
+	if refCategory.HostIPC >= containerSCC.HostIPC {
+		tnf.ClaimFilePrintf("HostIPC = %s - OK", containerSCC.HostIPC)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("HostIPC = %s but expected <= %s - NOK", containerSCC.HostIPC, refCategory.HostIPC)
+	}
+	// HostNetwork is true if the HostNetwork field is set to true, false otherwise.
+	if refCategory.HostNetwork >= containerSCC.HostNetwork {
+		tnf.ClaimFilePrintf("HostNetwork = %s - OK", containerSCC.HostNetwork)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("HostNetwork = %s but expected <= %s - NOK", containerSCC.HostNetwork, refCategory.HostNetwork)
+	}
+	// HostPID is true if the HostPID field is set to true, false otherwise.
+	if refCategory.HostPID >= containerSCC.HostPID {
+		tnf.ClaimFilePrintf("HostPID = %s - OK", containerSCC.HostPID)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("HostPID = %s but expected <= %s - NOK", containerSCC.HostPID, refCategory.HostPID)
+	}
+	// HostPorts is true if the HostPorts field is set to true, false otherwise.
+	if refCategory.HostPorts >= containerSCC.HostPorts {
+		tnf.ClaimFilePrintf("HostPorts = %s - OK", containerSCC.HostPorts)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("HostPorts = %s but expected <= %s - NOK", containerSCC.HostPorts, refCategory.HostPorts)
+	}
+	// PrivilegeEscalation is true if the PrivilegeEscalation field is set to true, false otherwise.
+	if refCategory.PrivilegeEscalation >= containerSCC.PrivilegeEscalation {
+		tnf.ClaimFilePrintf("HostNetwork = %s - OK", containerSCC.HostNetwork)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("PrivilegeEscalation = %s but expected <= %s - NOK", containerSCC.PrivilegeEscalation, refCategory.PrivilegeEscalation)
+	}
+	// PrivilegedContainer is true if the PrivilegedContainer field is set to true, false otherwise.
+	if refCategory.PrivilegedContainer >= containerSCC.PrivilegedContainer {
+		tnf.ClaimFilePrintf("PrivilegedContainer = %s - OK", containerSCC.PrivilegedContainer)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("PrivilegedContainer = %s but expected <= %s - NOK", containerSCC.PrivilegedContainer, refCategory.PrivilegedContainer)
+	}
+	// ReadOnlyRootFilesystem is true if the ReadOnlyRootFilesystem field is set to true, false otherwise.
+	if refCategory.ReadOnlyRootFilesystem >= containerSCC.ReadOnlyRootFilesystem {
+		tnf.ClaimFilePrintf("ReadOnlyRootFilesystem = %s - OK", containerSCC.ReadOnlyRootFilesystem)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("ReadOnlyRootFilesystem = %s but expected <= %s - NOK", containerSCC.ReadOnlyRootFilesystem, refCategory.ReadOnlyRootFilesystem)
+	}
+	// SeLinuxContextPresent is true if the SeLinuxContext field is present and set to a value (e.g. not nil), false otherwise.
```

Comment:
// An SELinuxContext strategy of MustRunAs with no level set. Admission looks for the openshift.io/sa.scc.mcs annotation to populate the level.